resources/lang/id/authorization.php

```diff
@@ -0,0 +1,160 @@
+<?php
+
+/**
+ *    Copyright 2015-2017 ppy Pty. Ltd.
+ *
+ *    This file is part of osu!web. osu!web is distributed with the hope of
+ *    attracting more community contributions to the core ecosystem of osu!.
+ *
+ *    osu!web is free software: you can redistribute it and/or modify
+ *    it under the terms of the Affero GNU General Public License version 3
+ *    as published by the Free Software Foundation.
+ *
+ *    osu!web is distributed WITHOUT ANY WARRANTY; without even the implied
+ *    warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *    See the GNU Affero General Public License for more details.
+ *
+ *    You should have received a copy of the GNU Affero General Public License
+ *    along with osu!web.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+return [
+    'beatmap_discussion' => [
+        'destroy' => [
+            'is_hype' => 'Tidak dapat membatalkan hyping.',
+            'has_reply' => 'Tidak dapat menghapus diskusi dengan balasan',
+        ],
+        'nominate' => [
+            'exhausted' => 'Anda telah mencapai batas nominasi anda untuk hari ini, silakan coba lagi besok.',
+        ],
+        'resolve' => [
+            'not_owner' => 'Hanya pemilik topik dan beatmap yang dapat menyelesaikan diskusi.',
+        ],
+
+        'vote' => [
+            'limit_exceeded' => 'Harap tunggu sebentar sebelum memberikan lebih banyak suara.',
+            'owner' => 'Tidak dapat memilih di diskusi sendiri.',
+            'wrong_beatmapset_state' => 'Hanya dapat memberikan suara pada diskusi di beatmap pending',
+        ],
+    ],
+
+    'beatmap_discussion_post' => [
+        'edit' => [
+            'system_generated' => 'Posting yang dihasilkan secara otomatis tidak dapat disunting.',
+            'not_owner' => 'Hanya pemilik topik yang dapat menyunting posting.',
```

Comment:
Since "-ing" refers to an active word it should be removed and be turned into a noun (without the -ing) in order for it to make more sense in Indonesian language :

+            'system_generated' => 'Post yang dihasilkan secara otomatis tidak dapat disunting.',
+            'not_owner' => 'Hanya pemilik topik yang diperbolehkan untuk menyunting post.',